docker/consul-template.conf

```diff
@@ -1,9 +1,15 @@
 template {
     source = "/code/consul_config.py.ctmpl"
     destination = "/code/consul_config.py"
-    command = "sv hup uwsgi"
 }
 template {
     source = "/code/admin/config.sh.ctmpl"
     destination = "/code/admin/config.sh"
 }
+exec {
+    command = "sv hup uwsgi"
```

Comment:
this should be `uwsgi /etc/uwsgi/uwsgi.ini` instead of sv hup (the command that used to be in uwsgi/uwsgi.service)